From 078ee1b05436adc67ba6175345ebdfd5c0463195 Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz93@gmail.com>
Date: Mon, 20 Jun 2022 19:59:35 -0400
Subject: [PATCH] fix broken C std for gvc subproject

Historically, Meson didn't know how to set c_std for a subproject at
all. Meson 0.63.0 adds support for this, so the default_options kwarg to
subproject now begins to respect this.

budgie-desktop uses c11, but artificially sets c89 for the gvc
submodule. However, this does not actually work... the build then errors
out when trying to build
subprojects/gvc/test-audio-device-selection.p/test-audio-device-selection.c.o

Remove the faulty argument.

Fixes regression in commit 3d6d71c167dd9742cd072e7e6a3453c23ebb2658.
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index 763d5a1b..3c52b8f5 100644
--- a/meson.build
+++ b/meson.build
@@ -130,7 +130,6 @@ with_polkit = get_option('with-polkit')
 # Get gvc built before we do anything
 gvc = subproject('gvc',
     default_options: [
-        'c_std=c89',
         'static=false',
         'pkglibdir=@0@'.format(rpath_libdir),
         'package_name=' + meson.project_name(),
